### ISSUE TITLE
3bot to work on unstable (10.4)

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/jumpscaleX_threebot#443]
- [x] new machine on DO
- [x] jsx threebot to work
- [x] container should go to http://localhost:7020/  [blocked on: https://github.com/threefoldtech/jumpscaleX_threebot/issues/341]
- [x] / should redirect to admin panel for default website (create nginx location for that)

**Comment by waleedhammam** [comment]
blocked on: https://github.com/threefoldtech/jumpscaleX_core/issues/635

**Comment by waleedhammam** [comment]
container should go to http://localhost:7020/ , blocked by: https://github.com/threefoldtech/jumpscaleX_threebot/issues/341

**Comment by waleedhammam** [comment]
jsx threebot to work -> yields installation and repos from development not unstable

**Comment by xmonader** [comment]
@grimpy can you advice please on the portforward problem?

**Comment by xmonader** [comment]
for portforwards/callbacks of the 3bot connect auth please fix in https://github.com/threefoldtech/home/issues/586

**Comment by hossnys** [comment]
port 7020 => 443 is not exposed in jsx container after doing 
```
/tmp/jsx threebot -d 

```

```
root@jsx:~# docker ps -a
CONTAINER ID        IMAGE                 COMMAND             CREATED             STATUS              PORTS                                                                NAMES
ae0eed2676ef        threefoldtech/3bot2   "/sbin/my_init"     2 hours ago         Up 2 hours          0.0.0.0:9001->9001/udp, 0.0.0.0:9000->22/tcp, 0.0.0.0:7000->80/tcp   3bot

```

**Comment by waleedhammam** [comment]
working on exposing port 7020 => 443 to be available automatically after installation

**Comment by waleedhammam** [comment]
Commit: https://github.com/threefoldtech/jumpscaleX_core/commit/6278e12d523d13a1cac4df9eb8123f2fa00ee9aa

test at:

https: https://167.172.140.48:7020/
http: http://167.172.140.48:7000

**Comment by Pishoy** [comment]
got issue when i run 
1 - create new machine DO 
2 - do below steps 
```
     curl https://raw.githubusercontent.com/threefoldtech/jumpscaleX_core/unstable/install/jsx.py?$RANDOM > /tmp/jsx
    chmod +x /tmp/jsx
    /tmp/jsx threebot -d

```
3 - installation finished and redirect to container 
4 - checked container tmux i got below picture 
![image](https://user-images.githubusercontent.com/21987477/77425354-47515780-6ddb-11ea-8956-b6d0a9280080.png)

**Comment by abom** [comment]
@Pishoy This is a different issue related to sonic, can you please create another issue?

For this one, can you please verify that port 7020 is exposed, and you can access threebot through it and login to `/admin`?

**Comment by Pishoy** [comment]
yes port is exposed 
but i have to start threebot manaully ` JSX> j.servers.threebot.start(background=True)  ` which should start automatic !
- also http works but https does not as below 
- http
![image](https://user-images.githubusercontent.com/21987477/77428757-491e1980-6de1-11ea-83cb-3425d863268a.png)

- https
![image](https://user-images.githubusercontent.com/21987477/77428797-5935f900-6de1-11ea-84f6-9977b9a82e0d.png)

**Comment by waleedhammam** [comment]
> yes port is exposed
> but i have to start threebot manaully `JSX> j.servers.threebot.start(background=True) ` which should start automatic !
> 
> * also http works but https does not as below
> * http
>   ![image](https://user-images.githubusercontent.com/21987477/77428757-491e1980-6de1-11ea-83cb-3425d863268a.png)
> * https
>   ![image](https://user-images.githubusercontent.com/21987477/77428797-5935f900-6de1-11ea-84f6-9977b9a82e0d.png)

http is on 7000

**Comment by Pishoy** [comment]
verified after 3bot container is created, installation script redirect you to container shell and you need to run manually `root@3bot:/sandbox# 3bot start ` to start threebot server

**Comment by waleedhammam** [comment]
Please note https now is on 6000 to avoid conflicts
commit: https://github.com/threefoldtech/jumpscaleX_core/commit/d4ab5d126be069100951ce465bcf418dbf154391
